mne/channels/montage.py

```diff
@@ -266,7 +266,7 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         az = np.deg2rad(np.array([90. - h if a >= 0. else -90. - h
                                   for h, a in zip(horiz, az)]))
         pol = radius * np.pi
-        pos = _sph_to_cart(np.array([np.ones(len(az)) * 85., az, pol])).T
+        pos = _sph_to_cart(np.array([np.ones(len(az)) * 85., az, pol]).T)
```

Comment:
Maybe the array was 3x3, and the output values (which would have been very broken) never checked. Or maybe the lines were never tested?